apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -91,18 +91,25 @@ const SendMessageToSupportService = new GQLCustomSchema('SendMessageToSupportSer
                 const files = await Promise.all(filesPromises)
 
                 const residents = await Resident.getAll(context, { user: { id: user.id } })
+                const serviceConsumers = await ServiceConsumer.getAll(context, { resident: { id_in: residents.map(x => x.id) } })
 
-                const organizationsIds = residents.reduce((accumulatedData, resident) => {
-                    if (resident.organization && resident.organization.id) {
-                        return [...accumulatedData, resident.organization.id]
+                const residentsExtraInfo = []
+
+                for (const resident of residents) {
+                    const residentServiceConsumers = serviceConsumers.filter(x => x.resident.id === resident.id)
+                    const residentOrganization = await Organization.getOne(context, { id: resident.organization.id })
+
+                    const residentInfo = { address: resident.address, accountNumbers: null, organization: null }
+
+                    if (residentServiceConsumers) {
+                        residentInfo.accountNumbers = [...residentServiceConsumers.map(x => x.accountNumber)]
```

Comment:
`ServiceConsumer.organization` may differ from `Resident.organization`. Please add the service consumer's organization info near each account number. Yes, in most cases this organization will be the same as a resident organization, but it's ok.

Reply:
done